### ISSUE TITLE
New component suggestion -> InsetHeader

### ISSUE THREAD

**Issue by aress31** [siriwatknp/mui-treasury#817]
Hi @siriwatknp 

I hope you are well. I was just thinking that an InsetHeader component could be useful for the case where we need to have search filters in an app further down the normal header and that either remains fixed or relative.

Please let me know what you think of the idea.

Kind regards,
@aress31

**Comment by siriwatknp** [comment]
Agree, I have seen something like that in gitbook. will try to make it live.
![image](https://user-images.githubusercontent.com/18292247/82350951-2b0e1780-9a26-11ea-8d78-b993c60dc598.png)
